tests/integration/test_httpx.py

```diff
@@ -314,3 +319,19 @@ def test_redirect_wo_allow_redirects(do_request, mockbin, yml):
         assert response.status_code == 308
 
         assert cassette.play_count == 1
+
+@pytest.mark.online
+@pytest.mark.parametrize("url", ["https://github.com/kevin1024/vcrpy/issues/" + str(i) for i in range(3, 6)])
+def test_simple_stream_fetching(yml, url):
+    client = httpx.Client()
+    with vcr.use_cassette(yml):
+        with client.stream("GET", url):
+            pass
```

Comment:
Not reading the response here would be weird in end user code. How about this pattern:

```python
    with vcr.use_cassette(..........):  # recording
        ........
        expected_content = ..........

    with vcr.use_cassette(..........):  # playback
        actual_content = ................
        assert actual_content == expected_content
```

Would be similar to what this existing test does:

https://github.com/kevin1024/vcrpy/blob/c95b7264a217a5411bc025807d466936699234c1/tests/integration/test_httplib2.py#L34-L43